GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.ChatWithTranscriptModel.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 
+// swiftlint:disable file_length
```

Comment:
I think it would be better if instead of disabling the file length, some parts of this file move to another file to make it easier to navigate this one. When does it complain? For 1000 lines I think? That is quite a lot.

Reply:
You can move this whole extension https://github.com/salemove/ios-sdk-widgets/blob/4358d0781ae7a1cd738e82afc068774cae899551/GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.ChatWithTranscriptModel.swift#L255 to a new file. It's perfect

Reply:
Done